datafusion/src/logical_plan/plan.rs

```diff
@@ -356,13 +356,15 @@ pub enum Partitioning {
 /// after all children have been visited.
 ////
 /// To use, define a struct that implements this trait and then invoke
-/// "LogicalPlan::accept".
+/// [`LogicalPlan::accept`].
```

Comment:
Question: What this change does? better looking in the doc?

Reply:
It makes it an hyperlink in API docs :)